scripts/utils.js

```diff
@@ -0,0 +1,28 @@
+/* @flow */
+
+import fs from "fs-extra";
```

Comment:
Is it possible to just use the built in `fs` module in Node.js? https://nodejs.org/docs/latest-v16.x/api/fs.html

Or does fs-extra provide something that fs does not? Just trying to cut down on any additional dependencies.
```suggestion
import fs from "fs";
```

Reply:
I used `fs-extra` as it seems to be a common convention I've seen in our repos, but using `fs` instead seems to work fine!